psi4/src/psi4/libmints/eribase.cc

```diff
@@ -1921,6 +2019,19 @@ void TwoElectronInt::init_shell_pairs12() {
     pairs12_ = new ShellPair *[basis1()->nshell()];
     for (i = 0; i < basis1()->nshell(); ++i) pairs12_[i] = new ShellPair[basis2()->nshell()];
 
+    //  Store screened shell pairs in a vector of vectors
+    for (i = 0; i < basis1()->nshell(); ++i) {
+        std::vector<ShellPairScreen> row;
+        screenpairs12_.push_back(row);
+    }
+
```

Comment:
I think this entire for loop can be replaced with: `screenpairs12_.resize(basis1()->nshell());`

Reply:
Definitely, I replaced the loop with a single initialization statement that creates `screenpairs12_` with the correct number of elements